azkaban-web-server/src/main/java/cloudflow/servlets/Constants.java

```diff
@@ -0,0 +1,16 @@
+package cloudflow.servlets;
+
+public class Constants {
+
+  public static final String EXECUTION_ID_KEY = "execution_id";
+  public static final String START_TIME_KEY = "start_time";
+  public static final String END_TIME_KEY = "end_time";
+  public static final String STATUS_KEY = "status";
+  public static final String ATTEMPTS_KEY = "attempts";
+  public static final String ID_KEY = "id";
+
+
+
+  public static final String DEFAULT_500_ERROR_MESSAGE = "Ooops, something went terribly wrong!";
```

Comment:
Maybe make the default message a little less severe sounding, something like "Unexpected/Internal error..." ? I believe the intention is to use it sparingly only when we don't have a more specific exception to return to the user?

Reply:
I'll change the message.